src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderFixedTermRecall.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.*;
+import org.hibernate.annotations.BatchSize;
+import uk.gov.justice.hmpps.prison.api.model.OffenderSentenceAndOffences;
+import uk.gov.justice.hmpps.prison.api.model.OffenderSentenceTerm;
+import uk.gov.justice.hmpps.prison.api.model.ReturnToCustodyDate;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.util.List;
+
+@Getter
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(of = {"offenderBooking"}, callSuper = false)
```

Comment:
Is this right for the equals and hashcode?

Reply:
Yea good spot. It should be bookingId.